tools_docker/Debian_Uwsgi/supervisord_tracim.conf

```diff
@@ -40,7 +40,9 @@ environment=TRACIM_CONF_PATH=/etc/tracim/development.ini
 ; RQ database worker
 [program:tracim_rq_worker]
 directory=/tracim/backend/
-command=rq worker -q -w tracim_backend.lib.rq.worker.DatabaseWorker event
+; NOTE 2021-02-23 - S.G. queue names should stay the same as RqQueueName enum values
+; mail_sender is separate as it has its own worker (named tracim_mail_notifier, just above)
+command=rq worker -q -w tracim_backend.lib.rq.worker.DatabaseWorker event elasticsearch_indexer
```

Comment:
Can you check if something can be missing in documentation `/backend/doc` about this new worker ?

Reply:
It's not new per se (it has been added for TLMs), a rq worker needs to be explicitly given its queue names hence the change + the note here.

Reply:
But you're right, a part of the documentation is explaining how to launch this worker, I'll edit it.

Reply:
Done.